### ISSUE TITLE
[test] increase test coverage to 70%

### ISSUE THREAD

**Issue by a-b-r-o-w-n** [microsoft/BotFramework-Composer#3456]
This issue is meant to be a long running issue that tracks efforts to increase unit test code coverage.

At the time is was created, coverage sits around 49%. Let's target an increase of 20% for R10.

**Comment by beyackle** [comment]
I don't think this was meant to be closed yet - not until we get to 70%, right?

**Comment by cwhitten** [comment]
Oops

**Comment by boydc2014** [comment]
number at 23/9 is 55%.

**Comment by boydc2014** [comment]
As a review guidance, let's make sure new PR have tests.

**Comment by boydc2014** [comment]
I will look at bottle-neck from the coverage report.

@a-b-r-o-w-n  will set a bar for coverage for PR check in, maybe start with -0.5%?

**Comment by a-b-r-o-w-n** [comment]
I set the threshold in coveralls to fail the build if coverage drops more than 0.5% or overall coverage drops below our current 55%
